cmd/populator-controller/BUILD.bazel

```diff
@@ -0,0 +1,37 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
+
+go_binary(
+    name = "populator-controller",
+    embed = [":populator-controller_lib"],
+    visibility = ["//visibility:public"],
+)
+
+go_library(
+    name = "populator-controller_lib",
+    srcs = ["populator-controller.go"],
+    importpath = "github.com/konveyor/forklift-controller/cmd/populator-controller",
+    visibility = ["//visibility:private"],
+    deps = [
+        "//pkg/apis/forklift/v1beta1",
+        "//vendor/github.com/kubev2v/lib-volume-populator/populator-machinery",
+        "//vendor/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured",
+        "//vendor/k8s.io/apimachinery/pkg/runtime",
+        "//vendor/k8s.io/apimachinery/pkg/runtime/schema",
+    ],
+)
+
+load(
+    "@io_bazel_rules_docker//container:container.bzl",
+    "container_image",
+)
+
+container_image(
+    name = "populator-controller-image",
+    base = "@ubi8-minimal//image",
```

Comment:
Does it need the ubi8? 
(I have just bumped all bazel images to ubi9)

Reply:
i don't think it needs ubi8. ubi9 should fit.